packages/gatsby-plugin-manifest/src/__tests__/gatsby-ssr.js

```diff
@@ -36,7 +35,25 @@ describe(`gatsby-plugin-manifest`, () => {
   })
 
   it(`Adds "shortcut icon" and "manifest" links and "theme_color" meta tag to head`, () => {
-    onRenderBody(ssrArgs, { icon: true, theme_color: `#000000` })
+    onRenderBody(ssrArgs, {
+      icon: true,
+      theme_color: `#000000`,
+    })
+    expect(headComponents).toMatchSnapshot()
+  })
+
+  it(`Adds link favicon tag if "include_favicon" is set to true`, () => {
+    onRenderBody(ssrArgs, { icon: true, include_favicon: true })
+    expect(headComponents).toMatchSnapshot()
+  })
+
+  it(`Adds link favicon if "include_favicon" option as is not provided `, () => {
```

Comment:
Small nit on spelling:
```suggestion
  it(`Adds link favicon if "include_favicon" option is not provided`, () => {
```